src/main/java/ch/njol/skript/lang/function/SimpleJavaFunction.java

```diff
@@ -19,6 +20,11 @@ public SimpleJavaFunction(String name, Parameter<?>[] parameters, ClassInfo<T> r
 		super(name, parameters, returnType, single);
 	}
 
+	@Internal
```

Comment:
I think it's more common to only import `ApiStatus` and annotate as `@ApiStatus.Internal`